docs/source/model_design/risk_factors.rst

```diff
@@ -198,68 +198,115 @@ Confounding
+Much of epidemiology is concerned with establishing associations between exposures and health outcomes. This usually involves comparing representative groups from two populations that are as similar as possible in all respects except the exposure(s) under study. **Confounding occurs when an estimate of the association between an exposure and a health outcome is mixed up with the effect of another exposure on the same health outcome, and the two exposures are associated**. When present, it results in a biased estimate of the effect of exposure on the health outcome. The bias can be negative (resulting in underestimation of the exposure effect), or positive (overestimation of the exposure effect), or can even reverse the apparent direction of effect. Confounding should always be addressed when assessing causality and failure to account for confounding leads to incorrectly concluding that an effect or its magnitude is due to one, rather than another variable. 
 
-An example:
-To illustrate this, imagine that you are interested in knowing whether smoking (our risk factor of interest) influences the risk of coronary heart disease (CHD) in men aged 18 to 64 years. You conduct a cohort study with an exposed group of male smokers in this age group from the general population, and compare their risk of CHD after a number of years to that of an unexposed group of non-smoking males in the same age group. You would expect to find a higher incidence of CHD among smokers. However, males who smoke are also likely to have a higher alcohol intake, which also increases the risk of CHD. Thus, the effect that you observe for the smoking-CHD association is **mixed**up with the effect of the association between alcohol intake and CHD. Without taking account of differences in alcohol intake between smokers and non-smokers, the magnitude of the smoking-CHD association estimated from the study may be higher than the **true** value, leading to incorrect conclusions.
+In order for a factor to be a confounder (C), it must meet the following criteria:
 
-**Confounding occurs when an estimate of the association between an exposure and a disease is mixed up with the effect of another exposure on the same disease, and the two exposures are associated.** Failure to account for confounding leads to incorrectly concluding that the effect is due to one, rather than the other variable. 
+  1. Must be a risk factor, or the surrogate of a risk factor, for the outcome (O) of interest in those who are unexposed to the exposure (E) being investigated.
+  2. Must be correlated, positively or negatively, with the exposure (E). If the study population is stratified into exposed and unexposed groups, this means that (C) has a differential distribution in the two groups.
```

Comment:
this seems still a little misleading--i think from what ali knew and the docs nathaniel found, it sounds like confounding _has_ to be causal, so we might want to avoid saying 'correlated'

Reply:
When we calculate the summary exposure value, we are evaluating the impact of multiple risk factors on the same health outcome. Is confounding happens only if those factors are in causal relationship rather than simply correlated?

Reply:
I take this comment back, after further discussion!

Reply:
I thought summary exposure value is for one risk factor (that has many categories). so the SEV measures the effect of one risk-outcome pair (adjusted for confounders- either correlated or causally linked to the outcome? don't know how GBD handles confounders...might be good to find out!)